lib/ddtrace/configuration/components.rb

```diff
@@ -125,10 +125,16 @@ def shutdown!(replacement = nil)
 
         # Shutdown workers
         runtime_metrics.enabled = false
-        runtime_metrics.stop(true)
+        runtime_metrics.stop(true, close_metrics: false)
```

Comment:
Minor: Is this `enabled = false` needed? Wouldn't stop work even if `enabled` was still `true`?

Reply:
It doesn't seem like it's strictly required. This `enabled` is part of the worker pooling/threading/loop pattern, so it's tricky to be sure, but as far as I can reason, it's not actually needed to properly shut down `runtime_metrics` here.